src/common/utils/utils.ts

```diff
@@ -127,7 +127,7 @@ export function getBrowserName() {
 }
 
 export function isAllowedCurrentBrowser() {
-  return getBrowserName() === Browser.CHROME || window.navigator.brave || Browser.FIREFOX;
+  return [Browser.CHROME, window.navigator.brave, Browser.FIREFOX].includes(getBrowserName());
```

Comment:
For Brave's case, although it might work because `getBrowserName` returns "Chrome", `window.navigator.brave` is an object and not the string "Brave".